src/desktop-lite/install.sh

```diff
@@ -293,7 +293,7 @@ echo -e "\nSuccess!\n"
 EOF
 
 # Container ENTRYPOINT script
```

Comment:
How are the tests passing locally? 

From https://github.com/devcontainers/features/actions/runs/9719274381/job/27190519697?pr=1024#step:4:3497

It looks like these PR changes are failing the init script run. Can you investigate here? 
 
![image](https://github.com/user-attachments/assets/e99cc178-049e-4b2b-984e-c0c78dc3971c)